app/helpers/service_requests_helper.rb

```diff
@@ -78,13 +78,7 @@ def save_as_draft_button
   end
 
   def step_nav_button(text, color, url, inactive_link)
-    if inactive_link
-      content_tag(:div,
-        content_tag(:div, raw(text), class: "btn step-text")+
-        content_tag(:div, '', class: "right-arrow"),
-        class: "step-btn step-btn-#{color} disabled_steps"
-      )
-    else
+    if !inactive_link
```

Comment:
The `inactive_link` logic can be removed entirely because it only exists to disable the step links on the confirmation page. It's used in `confirmation.html.haml` and `_steps.html.haml`